### PR TITLE
[FSSDK-8503]: Update Github Issue Templates.

### PR DESCRIPTION
**Pull request by yasirfolio3**
## Summary
- Update Github Issue Templates

## Issues
- [FSSDK-8503](https://jira.sso.episerver.net/browse/FSSDK-8503)

## Preview
- Changes can be previewed at: https://github.com/yasirfolio3/javascript-sdk/issues/new/choose

### CODE REVIEW

**Review by mikechu-optimizely — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["Bug", "Needs Triage"]
```
Comment (mikechu-optimizely) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:4]:
```suggestion
labels: ["bug", "needs-triage"]
```

File: .github/ISSUE_TEMPLATE/ENHANCEMENT.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: "Enhancement"
```
Comment (mikechu-optimizely) [inline .github/ISSUE_TEMPLATE/ENHANCEMENT.yml:1]:
```suggestion
name: ✨Enhancement
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Requests
```
Comment (mikechu-optimizely) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name:  💡Feature Requests
```

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]
Some suggestions.

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**